content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md

```diff
@@ -57,37 +95,35 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
           **Tip:** This example uses 4 processes to build in parallel. The best number of processes to use depends on how many CPU cores your hardware has available. You can use `cat /proc/cpuinfo` to get information about your hardware's processor.
 
-      6. Set the environment variable `BOOST_ROOT` to point to the new `boost_1_67_0` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
+      6. Set the environment variable `BOOST_ROOT` to point to the new `boost_1_68_0` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
 
-              export BOOST_ROOT=/home/ubuntu/boost_1_67_0
+              export BOOST_ROOT=/home/my_user/boost_1_67_0
 
       7. Source your updated `.profile` file. For example:
 
               source ~/.profile
 
-5. From a working directory, get the `rippled` source code. The `master` branch has the latest released version.
+6. From a working directory, get the `rippled` source code. The `master` branch has the latest released version.
 
-        cd ~
         git clone https://github.com/ripple/rippled.git
         cd rippled
         git checkout master
 
-6. Check the commit log to be sure you're compiling the version you intend to. The most recent commit should be signed by a well-known Ripple developer and should set the version number to the latest released version. For example:
+7. Check the commit log to be sure you're compiling the version you intend to. The most recent commit should be signed by a well-known Ripple developer and should set the version number to the latest released version. For example:
 
         $ git log -1
```

Comment:
since we sign our version bumps (head of release), should we suggest adding `--show-signature` to the git log command line? I don't know if it helps instill confidence or not.

Reply:
Discussed with Mike. Leaving it off for now because the user must have access to the signer's public key and those keys aren't readily accessible right now. I will follow up with Nik to see if we can make those available. We can then add this option to the command line and give users a link to the location of the public keys of folks who can sign `rippled` releases.